ppd/ppd-test.c

```diff
@@ -82,48 +83,48 @@ static int valid_path(const char *keyword, const char *path, int errors,
 static int valid_utf8(const char *s);
 
 
-//'ppdTest()' - Test the correctness of PPD files.
+// 'ppdTest()' - Test the correctness of PPD files.
  
 
 cups_array_t *ppdTest(int ignore, int warn, char *rootdir,
```

Comment:
Could you have one line for each parameter and add a short comment for each parameter what it is for? See the other functions in libppd. Also these comments are used for auto-generation of documentation.